### DISCUSSION TITLE
Setting component title dynamically doesn't work if lazy loading is enabled

### DISCUSSION THREAD

**Discussion by PopovicV** [livewire/livewire#7451] [7. Bugs (archived)]
### Livewire version

3.2.1

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

```
#[Title('Component Title')] 
#[Lazy]
class TestComponent extends Component
{
    // ...
}
```

### Additional information

I'm not able to set page title dynamically if lazy load is enabled.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by rama-can** [comment]
hello, I have the same problem, is this problem **solved?**
```php

#[Layout('layouts.admin')] 
#[Title('Resource')] 
#[Lazy]
class Resource extends Component
{
    //
}
```

**Comment by milosb793** [comment]
Having the same problem, seems like not fixed yet
